web/modules/custom/itk_cookie_message/src/Form/ItkCookieMessageConfigForm.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * @file
+ * Configuration form for the Cookie message module.
+ */
+
+namespace Drupal\itk_cookie_message\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Language\LanguageInterface;
+
+/**
+ * Configuration form for the Cookie message module.
+ */
+class ItkCookieMessageConfigForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'itk_cookie_message_admin_settings';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return array('itk_cookie_message.settings');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $settings = _itk_cookie_message_get_settings();
+
+    $form['tabs'] = array(
+      '#type' => 'vertical_tabs',
+      '#weight' => 99,
+    );
+
+    $form['general'] = array(
+      '#type' => 'details',
+      '#title' => $this->t('General settings'),
+      '#group' => 'tabs',
+    );
+
+    $key = 'cookie_name';
+    $form['general'][$this->getKey($key)] = array(
```

Comment:
Why use the getKey here, it's not language dependent and will allways be the same?